src/test/java/seedu/address/testutil/TypicalRooms.java

```diff
@@ -43,29 +41,22 @@
         .build();
 
     public static final Room SINGLE_023 = new RoomBuilder()
-            .withRoomNumber(TypicalRoomNumbers.ROOM_NUMBER_023.toString()).withCapacity(CAPACITY_SINGLE_ROOM)
-            .build();
+        .withRoomNumber(TypicalRoomNumbers.ROOM_NUMBER_023.toString()).withCapacity(CAPACITY_SINGLE_ROOM)
+        .build();
 
     public static final Room DOUBLE_024 = new RoomBuilder()
-            .withRoomNumber(TypicalRoomNumbers.ROOM_NUMBER_024.toString()).withCapacity(CAPACITY_DOUBLE_ROOM)
-            .build();
+        .withRoomNumber(TypicalRoomNumbers.ROOM_NUMBER_024.toString()).withCapacity(CAPACITY_DOUBLE_ROOM)
+        .build();
```

Comment:
Actually, indentation for wrapped lines should be 8 spaces.

https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html Layout point 2.

Reply:
Noted.